modules/bbp-simulation/src/main/resources/schemas/bbp/simulation/analysis/v0.1.0.json

```diff
@@ -3,7 +3,7 @@
     "{{base}}/contexts/nexus/core/schema/v0.1.0",
     {
       "nsg": "https://bbp-nexus.epfl.ch/vocabs/bbp/neurosciencegraph/core/v0.1.0/",
-      "this": "{{base}}/schemas/hbphackathon/simulation/analysis/v0.1.0/shapes/"
+      "this": "{{base}}/schemas/bbp/simulation/analysis/v0.1.0/shapes/"
```

Comment:
Would you want to start using "@base" instead of "this" in this PR already or can this wait until a later iteratiom?

Reply:
I prefer to delay it a bit in another iteration.